### DISCUSSION TITLE
How to clear the comments in the template after packaging?

### DISCUSSION THREAD

**Discussion by parlay96** [vitejs/vite-plugin-vue#450] [Q&A] [answered]
### Related plugins

- [X] [plugin-vue](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue)

- [ ] [plugin-vue-jsx](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue-jsx)

### Describe the bug

How to clear the comments in the template after packaging?
![4{4ZJ0PBD@LCDSJ (S%GCN](https://github.com/vitejs/vite-plugin-vue/assets/32978177/e8581628-bb68-42ca-8b8b-9bae1daf6f68)
![PXUPU4JT XLXF8(15FC0{}O](https://github.com/vitejs/vite-plugin-vue/assets/32978177/39c1c143-e2d0-4fa5-9866-3dc699c47115)

### Reproduction

no

### Steps to reproduce

![PXUPU4JT XLXF8(15FC0{}O](https://github.com/vitejs/vite-plugin-vue/assets/32978177/e54de20c-9958-4035-9a2c-2ea9b20215a0)

### System Info

```shell
win11
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitejs/vite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitejs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitejs/vite-plugin-vue/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Vite issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to [vuejs/core](https://github.com/vuejs/core) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitejs/vite-plugin-vue/discussions) or join our [Discord Chat Server](https://chat.vitejs.dev/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by junxiongchu** [comment, marked as answer]
try setting it like this：
vue({ template: { compilerOptions: { comments: false } } })
